homeassistant/components/smartthings/binary_sensor.py

```diff
@@ -134,6 +134,14 @@ class SmartThingsBinarySensorEntityDescription(BinarySensorEntityDescription):
             is_on_key="wet",
         )
     },
+    Capability.SAMSUNG_CE_DOOR_STATE: {
+        Attribute.DOOR_STATE: SmartThingsBinarySensorEntityDescription(
+            key=Attribute.DOOR_STATE,
+            translation_key="door",
+            device_class=BinarySensorDeviceClass.OPENING,
```

Comment:
Instead of adding a new `door` translation key, can't `BinarySensorDeviceClass.DOOR` be used directly? That's what Home Connect uses for smart appliances, see: [.../home_connect/binary_sensor.py#L99-L152](https://github.com/home-assistant/core/blob/ddd67a7e58c2a13d6126e8542ac8eaa74cbf6a25/homeassistant/components/home_connect/binary_sensor.py#L99-L152)
(Then the translation key and `[%key:component::binary_sensor::entity_component::door::name%]` part in `strings.json` could be removed entirely.)

Or was this done to avoid having the door icon for smart appliances that looks like a "house door"?

Reply:
That was indeed the main reason, as in theory its not a door how we know it. In theory it could als be a door, but I'm open for suggestions